File: src/mb/mizinkobusters/kitpvp/other/ArrowsRemover.java
package mb.mizinkobusters.kitpvp.other;

import org.bukkit.entity.EntityType;
import org.bukkit.entity.Projectile;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.entity.ProjectileHitEvent;

public class ArrowsRemover implements Listener {

    @EventHandler
    public void onHitArrow(ProjectileHitEvent event) {
        Projectile projectile = event.getEntity();
        if(!projectile.getType().equals(EntityType.ARROW)) {
            return;
        }
        if (!(projectile.getWorld().getName().equals("kitpvp")
                || projectile.getWorld().getName().equals("kitpvp2"))) {
            return;
        }
        projectile.remove();
    }
}

File: src/main/java/mb/mizinkobusters/kitpvp/other/ArrowsRemover.kt
package mb.mizinkobusters.kitpvp.other

import org.bukkit.entity.EntityType
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.entity.ProjectileHitEvent

class ArrowsRemover : Listener {
    @EventHandler
    fun onHitArrow(event: ProjectileHitEvent) {
        val projectile = event.entity
        if (projectile.type != EntityType.ARROW) {
            return
        }
        if (!(projectile.world.name == "kitpvp" || projectile.world.name == "kitpvp2")) {
            return
        }
        projectile.remove()
    }
}